tests/core/full_node/test_mempool.py

```diff
@@ -277,17 +280,17 @@ async def test_basic_mempool_manager(self, bt, two_nodes_mempool, wallet_a, self
             (ConditionOpcode.ASSERT_HEIGHT_RELATIVE, 0, MempoolInclusionStatus.PENDING),
             (ConditionOpcode.ASSERT_HEIGHT_RELATIVE, 1, MempoolInclusionStatus.PENDING),
             # the absolute height and seconds tests require fresh full nodes to
-            # run the test on. Right now, we just launch two simulations and all
-            # tests use the same ones. See comment at the two_nodes_mempool fixture
-            # (ConditionOpcode.ASSERT_HEIGHT_ABSOLUTE, 2, MempoolInclusionStatus.SUCCESS),
-            # (ConditionOpcode.ASSERT_HEIGHT_ABSOLUTE, 3, MempoolInclusionStatus.SUCCESS),
-            # (ConditionOpcode.ASSERT_HEIGHT_ABSOLUTE, 4, MempoolInclusionStatus.PENDING),
-            # (ConditionOpcode.ASSERT_HEIGHT_ABSOLUTE, 5, MempoolInclusionStatus.PENDING),
+            # run the test on. The fixture (two_nodes_mempool) creates 3 blocks,
+            # then condition_tester2 creates another 3 blocks
+            (ConditionOpcode.ASSERT_HEIGHT_ABSOLUTE, 4, MempoolInclusionStatus.SUCCESS),
+            (ConditionOpcode.ASSERT_HEIGHT_ABSOLUTE, 5, MempoolInclusionStatus.SUCCESS),
+            (ConditionOpcode.ASSERT_HEIGHT_ABSOLUTE, 6, MempoolInclusionStatus.PENDING),
```

Comment:
@mariano54  Is `PENDING` the correct return for `ASSERT_HEIGHT_ABSOLUTE 6` when chain height is 6?

Reply:
note that this is testing the mempool. When inserting a spend bundle in the mempool, conditions that *will hold* in the future are still saved, in a separate *pending* queue. That's what the `PENDING` return means.
Testing the consensus (which is done in a different test) these same tests are already enabled.

Reply:
In a funny coincidence, I just made a change that involved `MempoolInclusionStatus`, so I now understand this section better.